batch_gen.sh

```diff
@@ -19,27 +19,29 @@ getrand1(){
   #n=$? 
 }
 
-if [ "$#" -ne 2 ] || [ $2 -le 0 ] || ! [ -d "$1" ]; then
+if [ "$#" -ne 2 ] || [ $2 -le 0 ] ; then
   echo "usage: $0 <output dir> <#json files needed>" >&2
   exit 1
 fi
 
 path=$1
 nfiles=$2
+mkdir -p $path
```

Comment:
Unm, just asking. Why isn't this part of the python lib? can't it can be another command of the generate_network typer cli?

Reply:
could do, but I kinda wanted to expand on it and use it to test the impl., so thought it makes sense to separate it.